src/components/dock/dock-button/dock-button.tsx

```diff
@@ -124,14 +124,22 @@ export class DockButton {
                     selected: this.item?.selected,
                 }}
                 onClick={handleClick}
+                aria-live={this.item.badge ? 'polite' : 'off'}
```

Comment:
```suggestion
                aria-live='polite'
```

Reply:
I read the guidelines again and how I understand it this time is that we should always have the `polite` mode ON. Since we are telling the assistive technology **_in advance_** that this area can get a live update. Announce it properly, when it is updated.

Previously I thought once it gets the `polite` activated, the AT will see this immediately and properly and non-intrusively announces it to the user. Which was why I wrote the check the way I wrote it.

However, it seems to be unnecessary (if I understand the guidelines correctly).

Reply:
But I'm not sure we want to communicate `aria-live='polite'` about _all_ dock buttons?

My idea was that we would use `this.item.badge: false` or `this.item.badge: 0` to indicate that a badge might be shown later.

Reply:
Ok, let's just do `aria-live='polite'`. I'm ok with that.